ceph.spec.in

```diff
@@ -1058,6 +1060,15 @@ collecting data from Ceph Manager "prometheus" module and Prometheus
 project "node_exporter" module. The dashboards are designed to be
 integrated with the Ceph Manager Dashboard web UI.
 
+%if 0%{?suse_version}
+%package prometheus-alerts
+Summary:        Prometheus alerts for a Ceph deplyoment
+BuildArch:      noarch
+Group:          System/Monitoring
+%description
```

Comment:
This line needs to be `%description prometheus-alerts` for the build to succeed. Without it, we get:

```
[   80s] warning: line 1070: second Description
[   81s] error: Package has no %description: ceph-prometheus-alerts
```